README.rst

```diff
@@ -1,13 +1,16 @@
 Distributed
 ===========
 
-|Test Status| |Coverage| |Doc Status| |Discourse| |Version Status| |NumFOCUS|
+|Test Status| |Longitudinal Report| |Coverage| |Doc Status| |Discourse| |Version Status| |NumFOCUS|
 
 A library for distributed computation.  See documentation_ for more details.
 
 .. _documentation: https://distributed.dask.org
 .. |Test Status| image:: https://github.com/dask/distributed/workflows/Tests/badge.svg?branch=main
    :target: https://github.com/dask/distributed/actions?query=workflow%3A%22Tests%22
+.. |Longitudinal Report| image:: https://img.shields.io/badge/tests-longitudinal%20report-yellowgreen
+   :target: https://dask.github.io/distributed/test_report.html
```

Comment:
Oh nice! I like that this is in a more specific URL than https://dask.github.io/distributed